modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsUniqueDate.testcase

```diff
@@ -21,6 +21,71 @@ definition {
 		}
 	}
 
+	@description = "LPS-135398 - Verify unique values are enforced when editing an existing entry"
+	@priority = 5
+	test CanPreventDuplicateIntegerWhenEditing {
+		property custom.properties = "feature.flag.LPS-135398=true";
+		property portal.acceptance = "true";
+
+		task ("Given a custom object with a integer field") {
```

Comment:
Use the `task ("Given a Custom Object that contains an integer field with unique values only')`